context.go

```diff
@@ -641,15 +652,33 @@ func (c *Context) WalkBlueprintsFiles(rootFile string, visitor FileVisitor) (dep
 			c.parseOneCh(blueprint.string, blueprint.Scope, rootDir,
 				errsCh, fileCh, blueprintsCh, depsCh)
 
+			descendants, hasDescendants := descendantsMap[blueprint.string]
+			if hasDescendants {
+				for _, descendant := range descendants {
+					blueprintsCh <- stringAndScope{descendant, parser.NewScope(blueprint.Scope)}
```

Comment:
I think it makes more sense for the outer goroutine to handle the descendants.  I suggest removing blueprintCh, and returning the blueprint struct on doneCh.

Reply:
Does this mean to drop support for "subdirs="?

Reply:
I guess leave blueprintCh, but also return blueprint on doneCh so the outer one can handle descendants?  Or leave it this way if you think its cleaner.

Reply:
done